notifikasjon/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/HentDataTilSakOgOppgaveServiceTest.kt

```diff
@@ -112,7 +112,7 @@ private object Mock {
     val fnr = Fnr.genererGyldig()
     val orgnrMedNavn = mapOf(orgnr to "Kåre Conradis Kål og Kålrabi")
     val personer = mapOf(fnr to Person(fnr, "Kåre Conradi"))
-    val skalHaPaaminnelse = false // TODO: Bytt ut default false
+    val skalHaPaaminnelse = true
```

Comment:
Trenger vi variabelen?

Reply:
Tja, den brukes to steder så, tenker det får være greit.

Reply:
👍 Så bare én.